SpatialGDK/Build/Programs/Improbable.Unreal.Scripts/DeploymentLauncher/DeploymentLauncher.cs

```diff
@@ -0,0 +1,476 @@
+                {
+                    // Deployment not active - skip
+                    return false;
+                }
+
+                if (!deployment.Tag.Contains(DEPLOYMENT_LAUNCHED_BY_LAUNCHER_TAG))
+                {
+                    // Deployment not launched by this launcher - skip
+                    return false;
+                }
+
+                return true;
+            });
+        }
+
+        private static void ShowUsage()
+        {
+            Console.WriteLine("Usage:");
+            Console.WriteLine("DeploymentLauncher create <project-name> <assembly-name> <main-deployment-name> <main-deployment-json> <main-deployment-snapshot> <main-deployment-region> [<sim-deployment-name> <sim-deployment-json> <sim-deployment-region> <num-sim-players>]");
+            Console.WriteLine($"  Starts a cloud deployment, with optionally a simulated player deployment. The deployments can be started in different regions ('EU', 'US' and 'AP').");
+            Console.WriteLine("DeploymentLauncher stop <project-name> [deployment-id]");
+            Console.WriteLine("  Stops the specified deployment within the project.");
+            Console.WriteLine("  If no deployment id argument is specified, all active deployments started by the deployment launcher in the project will be stopped.");
+            Console.WriteLine("DeploymentLauncher list <project-name>");
+            Console.WriteLine("  Lists all active deployments within the specified project that are started by the deployment launcher.");
+        }
+
+        private static int Main(string[] args)
+        {
+            if (args.Length == 0 ||
+                args[0] == "create" && (args.Length != 11 && args.Length != 7) ||
+                args[0] == "stop" && (args.Length != 2 && args.Length != 3) ||
+                args[0] == "list" && args.Length != 2)
+            {
+                ShowUsage();
+                return 1;
+            }
+
+            try
+            {
+                if (args[0] == "create")
+                {
+                    return CreateDeployment(args);
+                }
+
+                if (args[0] == "stop")
+                {
+                    return StopDeployments(args);
+                }
+
+                if (args[0] == "list")
+                {
+                    return ListDeployments(args);
+                }
+
+                ShowUsage();
+            }
+            catch (Grpc.Core.RpcException e)
+            {
+                if (e.Status.StatusCode == Grpc.Core.StatusCode.Unauthenticated)
+                {
+                    Console.WriteLine("Error: unauthenticated. Please run `spatial auth login`");
+                }
+                else
+                {
+                    Console.Error.WriteLine($"Encountered an unknown gRPC error. Exception = {e.ToString()}");
+                }
+            }
+
+            return 1;
```

Comment:
Shouldn't this return 0?

Reply:
This is only reached if an error occurred or an unknown command was passed. Added a comment to clarify this